lib/src/core.dart

```diff
@@ -51,11 +51,17 @@ class Geometry {
       : null;
 
   Map<String, dynamic> toJson() {
-    var map = {};
+    var map = <String, dynamic>{};
     map['location'] = (location != null) ? location.toJson() : null;
-    map['location_type'] = locationType;
-    map['viewport'] = (viewport != null) ? viewport.toJson() : null;
-    map['bounds'] = (bounds != null) ? bounds.toJson() : null;
+    if (locationType != null) {
```

Comment:
maybe we should do the same for every `toJson` methods and avoid sending null value

Reply:
Can you at least do it for every `toJson` method you modified.

We can patch other `toJson` method in a second PR

Reply:
Or maybe it's time to use json_serializable ...